pkg/device/device.go

```diff
@@ -22,23 +22,23 @@ import (
 )
 
 const (
-	Nch = 64
```

Comment:
Please, do not touch lines irrelevant to the issue you are trying to solve. This is the strict requirement.

Reply:
But I dont!

Reply:
How on Earth you don't if there are plenty of lines changed?